packages/components/src/form-token-field/style.scss

```diff
@@ -102,7 +102,6 @@
 		&.is-error {
 			.components-form-token-field__token-text {
 				color: $alert-red;
-				border-radius: 4px 0 0 4px;
```

Comment:
I think this is safe to remove. The radius here is never visible:

<img width="154" alt="Screenshot 2024-08-22 at 10 56 32" src="https://github.com/user-attachments/assets/e3a9c5cd-0055-4504-9675-27441a0bc6f6">

Reply:
Agreed — also, I don't see why there should be a different radius (4px) compared to the standard (2px)